src/esp/assets/CMakeLists.txt

```diff
@@ -54,6 +54,11 @@ endif()
 
 add_library(assets STATIC ${assets_SOURCES})
 
+if(MSVC)
+  # Define common math constants if we compile with MSVC.  This is needed because of Sophus.
+  target_compile_definitions(assets PUBLIC _USE_MATH_DEFINES)
+endif (MSVC)
```

Comment:
Just `endif()`, repeating the condition is [considered a bad practice nowadays](https://cmake.org/cmake/help/latest/command/endif.html).

Same in the other two cases.